scripts/generate_rest.sh

```diff
@@ -1,11 +1,28 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
```

Comment:
Is there no update for the Powershell version?

Reply:
I've never done it before but: https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.4